lib/kino/process.ex

```diff
@@ -421,15 +430,37 @@ defmodule Kino.Process do
     {func_result, sequence_diagram}
   end
 
-  defp generate_participant_entry(pid, idx) do
+  defp get_label(pid) do
+    # :proc_lib.get_label/1 was added in OTP 27
+    case function_exported?(:proc_lib, :get_label, 1) do
+      true -> :proc_lib.get_label(pid)
+      false -> :undefined
+    end
+  end
+
+  defp generate_participant_entry(pid, idx, process_label) do
     try do
       {:registered_name, name} = process_info(pid, :registered_name)
       "participant #{idx} AS #{module_or_atom_to_string(name)};"
     rescue
-      _ -> "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+      _ ->
+        case process_label do
+          :undefined ->
+            "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+
+          process_label ->
+            "participant #{idx} AS #{format_for_mermaid_participant_alias(pid, process_label)};"
+        end
     end
   end
 
+  defp format_for_mermaid_participant_alias(pid, process_label) do
+    %{"pid" => pid_text} = Regex.named_captures(~r/#PID(?<pid>.*)/, inspect(pid))
```

Comment:
Similar here, you can use :erlang.pid_to_list

Reply:
or let's use `inspect(pid)` and that's it?

Looking at the screenshot in PR description, before:

```
#PID<0.2234.0>
```

After:

```
:ponger
<0.2234.0>
```

I think doing this:

```
:ponger
#PID<0.2234.0>
```

doesn't hurt and it is super clear what the number is. Unless I'm missing something?

Reply:
Repeating PID every time is verbose IMO.